package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "Symphony",
-  "productName": "Symphony",
+  "productName": "Symphony - ODP",
```

Comment:
The ODP is the Foundation's [Open Developer Platform](https://symphonyoss.atlassian.net/wiki/display/FM/Open+Developer+Platform).